de_workloads/data_processing/silver/silver.yml

```diff
@@ -173,6 +167,7 @@ stages:
                   RunAsPreJob: false
              displayName: 'Get secrets from the keyvault'
 
+          # TODO: Make the Databricks / spark steps conditional if data quality is not required
```

Comment:
For Silver this is not conditional, right?

Reply:
removed